apps/blog-app/vite.config.ts

```diff
@@ -26,6 +26,9 @@ export default defineConfig(() => {
               '/blog/2022-12-31-my-second-post',
             ];
           },
+          sitemap: {
+            domain: 'example.com/',
```

Comment:
nit: use the deployment URL

```suggestion
            domain: 'https://analog-blog.netlify.app/',
```